### PR TITLE
chore: update dependencies

### PR DESCRIPTION
**Pull request by MagneH**
- add missing dev dependencies
- update dependencies
- update some deprecated imports and config objects
- format according to defined project rules

### CODE REVIEW

File: web/package.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
     "build": "gatsby build",
     "deploy": "npm run now-deploy",
     "develop": "gatsby develop",
+    "empty-cache": "rimraf .cache",
```
Comment (kmelve) [inline web/package.json]:
```suggestion
    "empty-cache": "gatsby clean",
```

Comment (kmelve) [reply web/package.json]: This probably eliminates `rimraf` as a necessary dependency.

**Comment by MagneH** [comment]
> Great work - thanks! Only one suggestion to replace `rimraf` with the new `gatsby clean` that does the same.

I agree! Removed rimraf from the project, seems cleaner!

File: web/package.json (hunk 1 above)
Comment (MagneH) [reply web/package.json]: I agree! Removed `rimraf` from the project, seems cleaner!

**Pull request merged**